maxim/logger/groq/client.py

```
@@ -0,0 +1,229 @@
+                    "model": model,
+                    "provider": "groq",
+                    "name": generation_name,
+                    "model_parameters": GroqUtils.get_model_params(**kwargs),
+                    "messages": GroqUtils.parse_message_param(messages),
+                }
+                generation = trace.generation(gen_config)
+                
+                # Check for image URLs in messages and add as attachments
+                GroqUtils.add_image_attachments_from_messages(generation, messages)
+                        
+            except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)